contents/tutorials/android-session-replays.md

```diff
@@ -0,0 +1,262 @@
+---
+title: How to set up Android session replays
+date: 2024-07-10
+author:
+  - lior-neu-ner
+tags:
+  - session replay
+---
+
+[Session replays](/session-replay) are a useful support tool for understanding how users are interacting with your Android app. It also helps you debug and recreate issues. 
+**
```

Comment:
Whats the intention here? seems like a typo somewhere, its not breaking a line at least

Reply:
```suggestion

```